src/tensorlake/builder/client_v2.py

```diff
@@ -195,8 +196,25 @@ async def build(self, context: BuildContext, image: Image) -> BuildInfo:
         res.raise_for_status()
         build = BuildInfo.model_validate(res.json())
 
+        click.secho(f"Starting build for image {image.image_name} ...", fg="green")
         click.secho(f"Build ID: {build.id}", fg="green")
-        return await self.stream_logs(build)
+        try:
+            return await self.stream_logs(build)
+        except (asyncio.CancelledError, KeyboardInterrupt, click.Abort):
+            try:
```

Comment:
Would be easier to read if this whole try block under except(KeyboardInterrupt) was moved into self._cancel_build().

I mean try inside except is not something we see often.